docs/chi-examples/06-advanced-layout-01-one-shard-many-replicas.yaml

```diff
@@ -21,10 +21,10 @@ spec:
         spec:
           containers:
             - name: clickhouse-pod
-              image: clickhouse/clickhouse-server:22.1
+              image: clickhouse/clickhouse-server:23.8
```

Comment:
sync template name with and ferenece to template name in line 13 and line 20 above